tests/plugins/test_koji.py

```diff
@@ -114,8 +123,16 @@ class TestKoji(object):
         ('https://nosuchwebsiteforsure.com',
          False,
          'sslverify=0',
+         None,
          None),
 
+        # https with no cert available
+        ('https://nosuchwebsiteforsure.com',
```

Comment:
Maybe we could flexmock() out the actual certificate fetching, so we aren't hitting the network for this test?